index.js

```diff
@@ -4,7 +4,134 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+ 
+
+function calculator() {
+  
+
+    if (!process.argv[2]) {
+      return "No operation provided...";
+     
+    } 
+  
+    if (!process.argv[3]){
+      return "No numbers provided...";
+     } 
+    
+     if (process.argv[2] === "plus") {
```

Comment:
If you refactor this to have your conditional which checks for `plus` or `minus` inside the for loop, you can avoid repeating code.